Formula/gitsome.rb

```diff
@@ -0,0 +1,43 @@
+class Gitsome < Formula
+  include Language::Python::Virtualenv
+
+  desc "Supercharged Git/GitHub command-line interface (CLI)"
+  homepage "https://github.com/donnemartin/gitsome"
+  url "https://github.com/donnemartin/gitsome/archive/0.8.0.tar.gz"
+  sha256 "6b0c9196d43c730b2d66ce39a78c25983661c2629ecc07f8bd50e92e25f8a841"
+
+  depends_on "freetype"
+  depends_on "jpeg"
+  depends_on "python"
+
+  resource "Pillow" do
+    url "https://files.pythonhosted.org/packages/5b/bb/cdc8086db1f15d0664dd22a62c69613cdc00f1dd430b5b19df1bea83f2a3/Pillow-6.2.1.tar.gz"
+    sha256 "bf4e972a88f8841d8fdc6db1a75e0f8d763e66e3754b03006cbc3854d89f1cb1"
+  end
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    system libexec/"bin/pip", "install", ".[all]"
```

Comment:
We really want to have dependencies listed as resources, and install them.

Reply:
I have same opinion with that, and I tried. But an error like the one I posted to issue of gitsome will occur.

Reply:
```
pkg_resources.DistributionNotFound: The 'uritemplate.py<4.0.0,>=1.0.0' distribution was not found and is required by gitsome
```

This means you need to include the right version of `uritemplate` as a resource. Look at https://github.com/tdsmith/homebrew-pypi-poet

Reply:
Hmm...

```rb
resource "uritemplate" do
  url "https://files.pythonhosted.org/packages/cd/db/f7b98cdc3f81513fb25d3cbe2501d621882ee81150b745cdd1363278c10a/uritemplate-3.0.0.tar.gz"
  sha256 "c02643cebe23fc8adb5e6becffe201185bf06c40bda5c0b4028a93f1527d011d"
end

resource "uritemplate.py" do
  url "https://files.pythonhosted.org/packages/12/97/e12695c7d7710143767022ce931061b4a6b5b19982b20ecf5d71cdde3da1/uritemplate.py-3.0.2.tar.gz"
  sha256 "e0cdeb0f55ec18e1580974e8017cd188549aacc2aba664ae756adb390b9d45b4"
end
```

Reply:
https://gist.github.com/imbsky/8f456ff469976edf8435336ea29b47d2

Reply:
https://gist.github.com/imbsky/f483f91b031c681ba54443db3ebc7cbc

Reply:
https://gist.github.com/imbsky/f483f91b031c681ba54443db3ebc7cbc

Reply:
Have you figured out why `uritemplate.py` is not actually found?

Reply:
I found a similar issue, but I don't know if it's related.
https://github.com/SFDO-Tooling/CumulusCI/issues/883

Reply:
I have identified the cause, probably I need to change the gitsome code a little. The difficulty is low, but if @donnemartin doesn't have much time, I and my friend can fix it.

Reply:
@donnemartin What do you think?